Authentication.md

```diff
@@ -8,7 +8,7 @@ The restriction for authorization only applies to access to the Command Server (
 
 ### Authentication
 
-You can login as any sudoer, but Change Password only allows changing the iiab_admin_user password as the user field is read only.
+You can login as any sudoer or member of the iiab_admin_user_group, but Change Password only allows changing the iiab_admin_user password as the user field is read only.
```

Comment:
Typo: I assume you mean `{{ iiab_admin_user_grp }}` not iiab_admin_user_group ?

Reply:
Or if this design in still under discussion, perhaps consider something shorter like {{ iiab_admin_group }} ?

(cc @jvonau who's reviewing possible designs here on iiab/iiab#2570)

Reply:
See merged PR #348

Reply:
See also precursors (#345, PR #346) and high-level summary https://github.com/iiab/iiab/blob/master/roles/iiab-admin/README.rst